executor/poolmgr/gpm.go

```diff
@@ -241,3 +248,59 @@ func (gpm *GenericPoolManager) IsValidPod(kubeObjects []apiv1.ObjectReference, p
 	}
 	return false
 }
+
+// idleObjectReaper reaps objects after certain idle time
+func (gpm *GenericPoolManager) idleObjectReaper(kubeClient *kubernetes.Clientset,
+	fissionClient *crd.FissionClient,
+	fsCache *fscache.FunctionServiceCache,
+	idlePodReapTime time.Duration) {
+
+	pollSleep := time.Duration(2 * time.Minute)
+	for {
```

Comment:
It looks like this loop cannot be stopped, maybe add a close channel or something like that, to allow this to be stopped gracefully.

Reply:
The loop will stop only when container exited just like other loops
https://github.com/fission/fission/blob/e977c7caca4167eb1c6d61792b6cd71a658675b2/executor/poolmgr/gpm.go#L112-L115
I think it should be enough.